docs/Developer Documentation/Logger/index.md

```diff
@@ -0,0 +1,163 @@
+# SDL Logger
+
+SDL Core uses log4cxx for logging.
```

Comment:
Might be good to add a link here
```suggestion
SDL Core uses [log4cxx](https://logging.apache.org/log4cxx/latest_stable/) for logging.
```

Reply:
Done in https://github.com/smartdevicelink/sdl_core_guides/pull/93/commits/1e1bbb0c0d0780f5643eec5945d41a35ddc0e786